pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers=[
   "Typing :: Typed",
 ]
 dependencies = [
-  "pulp-cli>=0.18.2,<0.25",
+  "pulp-cli>=0.18.2,<0.26",
```

Comment:
What is a sane minimum pulp-cli version to ask for?

Reply:
I guess I need at leas 0.20.0 if I want to use `specifier` for `PluginRequirement`.

Reply:
IDK. You can try bisecting to find the best one, or just pick 0.25.0.